scripts/remove_boilerplate.py

```diff
@@ -191,11 +204,36 @@ def parse_arguments():
     return args
 
 
+def good_token(token):
+    # A token is good if:
+    # * It is on the whitelist
+    # * It is a word ending in ...
+    # * It does not have 3 consecutive punctuation marks
+    global whitelist
+    if token in whitelist:
+        return True
+    matcher3dots = re.compile(r'^\w+\.\.\.$')
```

Comment:
Compile the regexs outside of the function. `compile` is not cheap, and you're doing it for each and every token like this.

Reply:
I wanted to that before finalizing the code, I just forgot, thx!